### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ivanbokiitest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# testrepo
```
Comment (ivanbokii) [inline README.md]: not sure about thsi thing

**Review by ivanbokii — DISMISSED** [review]
approving for now, but will discard later

**Review by ivanbokiiua — CHANGES_REQUESTED** [review]
this does not look right

**Review by ivanbokii — DISMISSED** [review]
I totally agree with @ivanbokiiua

**Review by ivanbokiiua — CHANGES_REQUESTED** [review]
this does not looks right right

**Review by ivanbokiiua — DISMISSED** [review]
asdfasdfasdfsdf

**Review by ivanbokii — DISMISSED** [review]
meh meh meh

**Review by ivanbokii — DISMISSED** [review]
something for everyone

**Review by ivanbokii — DISMISSED** [review]
om gom omg

**Review by ivanbokii — DISMISSED** [review]
this does not look okok

**Review by ivanbokii — DISMISSED** [review]
meh

**Review by ivanbokiiua — DISMISSED** [review]
oh yeah, this looks amazing

**Review by ivanbokiiua — CHANGES_REQUESTED** [review]
I don't really like this shit

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
\ No newline at end of file
+# testrepo
+
```
Comment (ivanbokii) [inline README.md:2]: oh no no no

**Pull request closed without merging**